### DISCUSSION TITLE
Understanding bounds and initial conditions in pyddm

### DISCUSSION THREAD

**Discussion by yanivabir** [mwshinn/PyDDM#52] [Ask for help] [answered]
Hi,

I'm a little confused about the meaning of the bounds and initial conditions, relative to correct/error and actual choice.

I see that pyddm does not care whether the participant chose option 1 or 2, only whether the choice was correct.

Indeed, when you sample from a model, you only get the correctness of a choice, not the actual choice.

Does that mean that the two bounds are a correct and error bound?
If so - what is the meaning of initial conditions? And is there a way to model a bias towards (say) choice 1, regardless of whether it is correct?

Many thanks,
Yaniv

**Comment by mwshinn** [comment, marked as answer]
Yes, the upper bound always represents correct responses and the bottom bound error.  You can make an InitialConditions which flips direction based on the task conditions.  See: https://pyddm.readthedocs.io/en/latest/cookbook/initialconditions.html#ic-biased

**Reply by yanivabir** [reply]
Thanks! that clears it up
